GUI/src/main/kotlin/logic/caches/ThumbnailCache.kt

```diff
@@ -1,6 +1,7 @@
 package logic.caches
 
 import androidx.compose.ui.graphics.ImageBitmap
+import org.bytedeco.opencv.opencv_core.Mutex
```

Comment:
shouldn't this be `import kotlinx.coroutines.sync.Mutex` ? Anton said something about this.